src/DIPS.Xamarin.UI/Controls/Tag/Tag.xaml.cs

```diff
@@ -0,0 +1,310 @@
+﻿using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace DIPS.Xamarin.UI.Controls.Tag
+{
+    /// <summary>
+    ///     A <c>Frame</c> wrapping a <c>Label</c> to allow setting border radius for the <c>Label</c>.
+    /// </summary>
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class Tag
+    {
+        /// <inheritdoc />
+        public Tag()
+        {
+            InitializeComponent();
+        }
+
+        #region Label Properties
+
+        /// <summary>
+        ///     Bindable property for <see cref="Text" />
+        /// </summary>
+        public static readonly BindableProperty TextProperty =
+            BindableProperty.Create(nameof(Text), typeof(string), typeof(Tag), Label.TextProperty.DefaultValue);
+
+        /// <summary>
+        ///     Bindable property for <see cref="TextColor" />
+        /// </summary>
+        public static readonly BindableProperty TextColorProperty =
+            BindableProperty.Create(nameof(TextColor), typeof(Color), typeof(Tag), Label.TextColorProperty.DefaultValue);
+
+        /// <summary>
+        ///     Bindable property for <see cref="FontFamily" />
+        /// </summary>
+        public static readonly BindableProperty FontFamilyProperty =
+            BindableProperty.Create(nameof(FontFamily), typeof(string), typeof(Tag), Label.FontFamilyProperty.DefaultValue);
+
+        /// <summary>
+        ///     Bindable property for <see cref="FontAttributes" />
+        /// </summary>
+        public static readonly BindableProperty FontAttributesProperty = BindableProperty.Create(nameof(FontAttributes),
+            typeof(FontAttributes), typeof(Tag), Label.FontAttributesProperty.DefaultValue);
+
+        /// <summary>
+        ///     Bindable property for <see cref="FontSize" />
+        /// </summary>
+        public static readonly BindableProperty FontSizeProperty =
+            BindableProperty.Create(nameof(FontSize), typeof(double), typeof(Tag), Label.FontSizeProperty.DefaultValue);
```

Comment:
Use Defaultvaluecreator instead of default value. I don't this will give you the default named font as default?

Reply:
```suggestion
            BindableProperty.Create(nameof(FontSize), typeof(double), typeof(Tag), defaultvaluecreator: Label.FontSizeProperty.DefaultValueCreator);
```

Reply:
`Label.FontSizeProperty.DefaultValue` is the default value the Xamarin Framework itself uses for the `Label.FontSize` property.

Can you give an example scenario where the **default named font** would be considered as the default value?

Reply:
In `Xamarin.Forms/Xamarin.Forms.Core/Label.cs`
<img width="760" alt="image" src="https://user-images.githubusercontent.com/8718745/75510945-8bd31880-5a12-11ea-8f7d-b5b35f54e3d4.png">

and `Xamarin.Forms/Xamarin.Forms.Core/FontElement.cs`

<img width="760" alt="image" src="https://user-images.githubusercontent.com/8718745/75510917-7958df00-5a12-11ea-945b-db4c42d636a9.png">

the default value the framework uses is `-1`

Reply:
it supplies the `defaultValueCreator` as a separate parameter

Reply:
I passed a `FontSizeDefaultValueCreator` for `defaultValueCreator` parameter